GNUmakefile

```diff
@@ -44,8 +44,13 @@ test: fmtcheck
     		xargs -t -n4 go test -tags "all" $(TESTARGS) -timeout=30s -parallel=4
 
 testacc: fmtcheck
+	@echo "==> Sourcing .env file if avaliable"
+	if [ -f .env ]; then set -o allexport; source .env; set +o allexport; fi
 	TF_ACC=1 go test -tags "all" $(TEST) -v $(TESTARGS) -timeout 120m
 
+testacc-skip-tests: fmtcheck
+	TF_ACC=1 SKIP_NOT_SET_TESTS=1 go test -tags "all" $(TEST) -v $(TESTARGS) -timeout 120m
```

Comment:
Why add `SKIP_NOT_SET_TESTS`? If this environment variable not set,  raise an panic if enough for users.

Reply:
You're right, as I've discovered these values can be set to be dummy values